src/modules/function.js

```diff
@@ -0,0 +1,73 @@
+import { addTask, updateLocalStorage } from './app.js';
+
+const removeTask = (tasks, taskId) => {
+  if (taskId >= 0 && taskId < tasks.length) {
+    tasks.splice(taskId, 1);
+    updateLocalStorage(tasks);
+  }
+};
+
+const clearCompletedTasks = (tasks) => {
+  const remainingTasks = tasks.filter((task) => !task.completed);
+  tasks.length = 0;
+  remainingTasks.forEach((task) => tasks.push(task));
+  updateLocalStorage(tasks);
+};
+
+export default function populateTodoList() {
+  const tasksString = localStorage.getItem('tasks');
+  const tasks = tasksString ? JSON.parse(tasksString) : [];
+
+  const todoList = document.getElementById('todo');
+
+  todoList.innerHTML = '';
```

Comment:
It is recommended to use createElement() and appendChild() methods to create and add elements to the DOM instead of using innerHTML, which can be a security risk if the content is not sanitized properly.